### PR TITLE
feat: add root stack description to the root stack in the initializer

### PR DESCRIPTION
**Pull request by kaustavghosh06**

### CODE REVIEW

File: packages/amplify-provider-awscloudformation/src/initializer.js [hunk 1]
```diff
@@ -36,10 +36,18 @@ async function run(context) {
     stackName = verifiedStackName;
     const authRoleName = `${stackName}-authRole`;
     const unauthRoleName = `${stackName}-unauthRole`;
+
+    let nestedStack = JSON.parse(fs.readFileSync(initTemplateFilePath).toString());
```
Comment (edwardfoyle) [inline packages/amplify-provider-awscloudformation/src/initializer.js:40]: use JSONUtilities from `amplify-cli-core`

File: packages/amplify-provider-awscloudformation/src/push-resources.js [hunk 2]
```diff
@@ -510,10 +510,9 @@ function uploadTemplateToS3(context, resourceDir, cfnFile, category, resourceNam
 function formNestedStack(context, projectDetails, categoryName, resourceName, serviceName, skipEnv) {
   /* eslint-enable */
   const initTemplateFilePath = path.join(__dirname, '..', 'resources', 'rootStackTemplate.json');
-  const nestedStack = context.amplify.readJsonFile(initTemplateFilePath);
-
+  let nestedStack = context.amplify.readJsonFile(initTemplateFilePath);
```
Comment (edwardfoyle) [inline packages/amplify-provider-awscloudformation/src/push-resources.js:513]: why can't this stay as a const?

File: packages/amplify-provider-awscloudformation/src/initializer.js (hunk 1 above)
Comment (kaustavghosh06) [reply packages/amplify-provider-awscloudformation/src/initializer.js]: Updated.

File: packages/amplify-provider-awscloudformation/src/push-resources.js (hunk 2 above)
Comment (kaustavghosh06) [reply packages/amplify-provider-awscloudformation/src/push-resources.js]: Updated.

**Review by edwardfoyle — APPROVED** [review]

**Pull request merged**